src/client/containers/formComponent/formComponent.js

```diff
@@ -2,58 +2,45 @@ import React, { Component } from 'react';
 import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import FormField from '../../components/formField/formField';
+import RenderFormFields from '../../components/renderFormFields/renderFormFields';
 import postUserData from '../../actions/postUserDataAction';
-import getUserData from '../../actions/getUserDataAction';
 import './formComponent.css';
 
 class FormComponent extends Component {
     state = {
-      userName: '',
+      username: '',
       email: '',
       password: '',
     };
 
```

Comment:
it is better to set the initial values as null

Reply:
done